spec_cleaner/__init__.py

```diff
@@ -35,51 +35,106 @@ def process_args(argv: List[str]) -> Dict[str, Any]:
+        '--no-curlification',
+        action='store_true',
+        help='do not convert variables bracketing (%%{macro}) and keep it as it was on the input',
+    )
     parser.add_argument(
-        '-m', '--minimal', action='store_true', help='run in minimal mode that does not do anything intrusive (ie. just sets the Copyright)'
+        '--no-copyright',
+        action='store_true',
+        help='do not include official SUSE copyright hear and just keep what is present',
     )
     parser.add_argument(
-        '--no-curlification', action='store_true', help='do not convert variables bracketing (%%{macro}) and keep it as it was on the input'
+        '--remove-groups', action='store_true', help='remove groups from the specfile.'
     )
-    parser.add_argument('--no-copyright', action='store_true', help='do not include official SUSE copyright hear and just keep what is present')
-    parser.add_argument('--remove-groups', action='store_true', help='remove groups from the specfile.')
     parser.add_argument(
         '--copyright-year',
         metavar='YYYY',
         type=int,
         default=datetime.now().year,
         help='year to insert into the copyright header when re-generating it',
     )
-    output_group.add_argument('-o', '--output', default='', help='specify the output file for the cleaned spec content.')
+    output_group.add_argument(
+        '-o',
+        '--output',
+        default='',
+        help='specify the output file for the cleaned spec content.',
+    )
     parser.add_argument(
         '-p',
         '--pkgconfig',
         action='store_true',
         help='convert dependencies to their pkgconfig() counterparts, requires bit more of cleanup in spec afterwards.',
     )
-    parser.add_argument('--perl', action='store_true', help='convert dependencies to their perl() counterparts, highly expansive use carefully')
     parser.add_argument(
-        '-t', '--tex', action='store_true', help='convert dependencies to their tex() counterparts, requires verification of the output afterwards.'
+        '--perl',
+        action='store_true',
+        help='convert dependencies to their perl() counterparts, highly expansive use carefully',
+    )
+    parser.add_argument(
+        '-t',
+        '--tex',
+        action='store_true',
+        help='convert dependencies to their tex() counterparts, requires verification of the output afterwards.',
+    )
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='version',
+        version=__version__,
+        help='show package version and exit',
+    )
+    parser.add_argument(
+        '-k',
+        '--keep-space',
+        action='store_true',
+        help='keep empty lines in preamble intact.',
     )
-    parser.add_argument('-v', '--version', action='version', version=__version__, help='show package version and exit')
-    parser.add_argument('-k', '--keep-space', action='store_true', help='keep empty lines in preamble intact.')
```

Comment:
here those one liners actually are more readable than what blacken does with it.

Reply:
Well, in this case, I think it depends on a reader :) I personally find the formatted version more readable.

Reply:
okay, i am quite fine with it being adjusted differently.

Also note the __init__ content should be moved like @Conan-Kudo did it in the rpmlint, i was just lazy to do so myself so far...

Reply:
ok, let's do the init changes in another PR.

Reply:
Once the formatting stuff has settled, I can certainly make a PR to do the same in spec-cleaner. I did it in rpmlint because it was breaking stuff until I fixed it.